README.md

```diff
@@ -66,43 +66,43 @@ T.get('account/verify_credentials', { skip_status: true })
 //
 //  retweet a tweet with id '343360866131001345'
 //
-T.post('statuses/retweet/:id', { id: '343360866131001345' }, function (err, data, response) {
+twit.post('statuses/retweet/:id', { id: '343360866131001345' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 //  destroy a tweet with id '343360866131001345'
 //
-T.post('statuses/destroy/:id', { id: '343360866131001345' }, function (err, data, response) {
+twit.post('statuses/destroy/:id', { id: '343360866131001345' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 // get `funny` twitter users
 //
-T.get('users/suggestions/:slug', { slug: 'funny' }, function (err, data, response) {
+twit.get('users/suggestions/:slug', { slug: 'funny' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 // post a tweet with media
 //
-var b64content = fs.readFileSync('/path/to/img', { encoding: 'base64' })
+const b64contentwit = fs.readFileSync('/path/to/img', { encoding: 'base64' })
```

Comment:
small typo: this should be `b64contentwit`

Reply:
Actually should be, `b64content`, I was to change  `var` to `const` only

Reply:
ah yes, sorry I meant to type `b64content`, this one should be changed to `b64content` like you mentioned :)